dataprofiler/profilers/profile_builder.py

```diff
@@ -2869,7 +2874,7 @@ def tqdm(level: set[int]) -> Generator[int, None, None]:
 
         # Generate pool and estimate datasize
         pool = None
-        if self.options.multiprocess.is_enabled:
+        if self.options.multiprocess.is_enabled and auto_multiprocess_toggle:
             est_data_size = data[:50000].memory_usage(index=False, deep=True).sum()
```

Comment:
outside scope of this PR, but just noticed and wondered if this 50000 should hardcoded?